test/db_test.py

```diff
@@ -71,91 +76,101 @@ def assertDateNearNow(date):
 
 	@before
 	def beforeAll():
-		cleanup()
+		with app.app_context():
+			cleanup()
 
 	@after
 	def afterAll():
-		cleanup()
+		with app.app_context():
+			cleanup()
 
 	@describe('Get User')
 	def getUser():
-
-		@beforeEach
-		def _beforeEach():
-			cleanup()
-			addTestUser()
-
-		@it('User fields persisted')
-		def fieldsPreserved():
-			user = db.getUser(test_name)
-			assert_that(user.get('name'), equal_to(test_name))
-			assert_that(user.get('pass_hash'), equal_to(test_hash))
-			assert_that(user.get('pass_salt'), equal_to(test_salt))
-			assert_that(user.get('email'), equal_to(test_email))
-
-		@it('None returned for non-existing user')
-		def noUserFound():
-			user = db.getUser('badname')
-			assert_that(user, none())
+		with app.app_context():
+			@beforeEach
+			def _beforeEach():
+				with app.app_context():
```

Comment:
You've got a `with app.app_context()` inside another `with app.app_context()`. Do we really need to do that?

Reply:
That one's a mistake. I need to fix that, the top one doesn't actually do anything from what I remember.